access/resource_ip_access_list.go

```diff
@@ -30,48 +33,86 @@ func ResourceIPAccessList() common.Resource {
 	})
 	return common.Resource{
 		Schema: s,
+		CanSkipReadAfterCreateAndUpdate: func(d *schema.ResourceData) bool {
+			//only skip read after create
+			return d.IsNewResource()
```

Comment:
But if we'll have Read after Update, then we may get inconsistent state - API docs explicitly say that it may take a few minutes to propagate.